satpy/readers/virr_l1b.py

```diff
@@ -0,0 +1,73 @@
+from datetime import datetime
+from satpy.readers.hdf5_utils import HDF5FileHandler
+from pyspectral.blackbody import blackbody_wn_rad2temp as rad2temp
+import numpy as np
+import xarray as xr
+import dask.array as da
+import logging
+
+
+class VIRR_L1B(HDF5FileHandler):
+    """VIRR_L1B reader."""
+
+    def __init__(self, filename, filename_info, filetype_info):
+        super(VIRR_L1B, self).__init__(filename, filename_info, filetype_info)
+        logging.debug('day/night flag for {0}: {1}'.format(filename, self['/attr/Day Or Night Flag']))
+        self.geolocation_prefix = filetype_info['geolocation_prefix']
+        self.platform_id = filename_info['platform_id']
+        self.l1b_prefix = 'Data/'
+        self.wave_number = 'Emissive_Centroid_Wave_Number'
+        if filename_info['platform_id'] == 'FY3B':
+            self.l1b_prefix = ''
+            self.wave_number = 'Emmisive_Centroid_Wave_Number'
```

Comment:
Is this a typo in the code or in the file? Isn't it "Emissive"?

Reply:
Both the file and the documents you sent me have "Emissive". Here's a quote from the document you sent me awhile back: "Note the unusual spelling in Emmisive_Centroid_Wave_Number for VIRR " at the bottom of page 5.

Reply:
What does `ncdump -h` say?

Reply:
"Emmisive_Centroid_Wave_Number"